packages/deployer/test/ens.js

```diff
@@ -20,7 +20,7 @@ let ganacheOptions,
 describe("ENS class", () => {
   before(() => {
     ganacheOptions = {
-      // note that when vmErrorsOnRPCResponse is true, eager instamine must be enabled (default)
+      // note that when vmErrorsOnRPCResponse is true, strict instamine must be enabled (default)
```

Comment:
`vmErrorsOnRPCResponse: true` only works with `instamine: "eager"` (which is the default)